tuned/plugins/plugin_acpi.py

```diff
@@ -0,0 +1,78 @@
+from . import base
+from .decorators import *
+import os
+import tuned.logs
+from tuned.consts import ACPI_DIR
+
+log = tuned.logs.get()
+
+
+class ACPIPlugin(base.Plugin):
+	"""
+	`acpi`::
+
+	Plugin for ACPI driver configuration.
+
+	`platform_profile`:::
+	The [option]`platform_profile` option allows to set
+	the ACPI platform profile sysfs attribute, which provides
+	a generic power/performance preference API for other drivers.
+	Multiple profiles can be specified, separated by `|`.
+	The first available profile is selected.
+	+
+	.Selecting a platform profile
+	====
+	----
+	[acpi]
+	platform_profile=balanced|low-power
+	----
+	Using this option, *TuneD* will try to set the platform profile
+	to `balanced`. If that fails, it will try to set it to `low-power`.
+	====
+	"""
+	def __init__(self, *args, **kwargs):
+		super(ACPIPlugin, self).__init__(*args, **kwargs)
+
+	@classmethod
+	def _get_config_options(cls):
+		return {"platform_profile": None}
+
+	def _instance_init(self, instance):
+		instance._has_static_tuning = True
+		instance._has_dynamic_tuning = False
+
+	def _instance_cleanup(self, instance):
+		pass
+
+	@classmethod
+	def _platform_profile_choices_path(cls):
+		return os.path.join(ACPI_DIR, "platform_profile_choices")
+
+	@classmethod
+	def _platform_profile_path(cls):
+		return os.path.join(ACPI_DIR, "platform_profile")
+
+	@command_set("platform_profile")
+	def _set_platform_profile(self, profiles, sim):
+		if sim:
+			return str(profiles)
```

Comment:
The default value verification does, in fact, have the [option to check alternatives](https://github.com/redhat-performance/tuned/blob/2652d7c29d32e1ec97d6dcb2ed2e424594058523/tuned/plugins/base.py#L547) and it's used, e.g.,  when verifying EPP in the CPU plugin. That being said, I do like the option you propose more.

As I said in the initial comment, it would be nice to refactor the setters with alternatives so they behave the same (or do you think that the behavior differs on purpose?).